code/MathSpamProtectorField.php

```diff
@@ -126,7 +133,8 @@ public static function correct_answer($answer){
 
 		$word = MathSpamProtectorField::digit_to_word($v1 + $v2);
 
-		return ($word == strtolower($answer) || ($v1 + $v2) == $answer);
+		$allow_numeric_answer = Config::inst()->get('MathSpamProtectorField', 'allow_numeric_answer');
+		return ($word == strtolower($answer) || ((($v1 + $v2) == $answer) and $allow_numeric_answer));
```

Comment:
please use `&&` instead of `and` as your operator

Also, your order of comparison is wrong for the second part as you want to check the boolean of `$allowed_numeric_answer` before wasting CPU cycles on whether they got it right!

Assigning the variable also has no material benefit here, IMO.